source/TLS.rst

```diff
@@ -59,71 +62,256 @@ The steps to install the *cert-manager* are the following:
 
 The following commands perform all the needed actions:
 
-::
-    
-    kubectl create namespace cert-manager
-    kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true
-    kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.7/deploy/manifests/cert-manager.yaml
+.. code:: bash
+
+   $ kubectl create namespace cert-manager
+   $ kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true
+   $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.7/deploy/manifests/cert-manager.yaml
```

Comment:
I don't like this. Either we say "here is the example on how to install cert-manager ..., please read more about the installation here (link)", or we say that to install cert-manager please refer to this doc.
We should mention only the steps which are relevant to the operators.